lib/eco/incparser/annotation.py

```diff
@@ -43,6 +43,18 @@ class ToolTip(Hint):
     pass
 
 
+class Eval(Hint):
```

Comment:
The hints are only placeholders (they are "hints" for the bits of Eco that do the visualisation, so that the rendering code knows which annotations to use and which to ignore). I expect this sort of thing could be done better, but this fits with the model of the code we already have, a large-ish refactoring is probably worth doing soon.